### ISSUE TITLE
Set default smoothing for ensemble.ts

### ISSUE THREAD

**Issue by ryankelly-uiuc** [PecanProject/pecan#582]
Currently defaults to `window=1`, i.e. no smoothing. Change to... 48 (daily, assuming half-hourly output)?

**Comment by mdietze** [comment]
It's not safe to assume that model outputs will always be 30 min. Some models only run daily (DALEC), monthly, or annually (LINKAGES). Met products might be 30 min (most Ameriflux) 60 min (some Ameriflux), 3 hr (NARR), 6 hr (NCEP), or coarser. I think it's safer to say that we want PEcAn to be generating ensemble forecasts at no finer than daily resolution. So that means that the code will need to detect the frequency rather than just change the hard coded default.

**Comment by ryankelly-uiuc** [comment]
Good idea. You can assign this to me, though it won't be a top priority at the moment.
